sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_train_labelled_model_async.py

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_train_labelled_model_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to train a model with labelled data. See sample_recognize_custom_forms_async.py
+    to recognize forms with your custom model.
+USAGE:
+    python sample_train_labelled_model_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer subscription key
+    3) BLOB_SAS_URL - The shared access signature (SAS) Url of your Azure Blob Storage container with your labelled data.
+                      See https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/quickstarts/label-tool#connect-to-the-sample-labeling-tool
+                      for more detailed descriptions on how to get it.
+"""
+
+import os
+import asyncio
+
+
+class TrainLabelledModelSampleAsync(object):
+
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+    blob_sas_url = os.environ["BLOB_SAS_URL"]
```

Comment:
I would change to container_sas_url for clarity

Reply:
changing here and in the readme